dependencies.gradle

```
@@ -33,6 +33,11 @@ dependencies {
     modCompileOnly(forge.ae2)
     modCompileOnly(forge.ae2wtlib)
 
+    // Create
+    modCompileOnly(forge.ponder)
+    modCompileOnly("${forge.create.get()}:slim")
```

you can do what our mixin dep does here, e.g.
```suggestion
    modCompileOnly(variantOf(forge.create) { classifier("slim") })
```

---

also apply this change to the runtime create dependency